Jumpscale/clients/blockchain/tfchain/schemas/reservation_vm.schema

```diff
@@ -0,0 +1,7 @@
+@url = tfchain.reservation.zos_vm
+type = "vm" (S)
+duration = (I)
```

Comment:
Just a remark, but does a schema not allow to add something later to it? E.g. a new property, so you only have to add the duration property when you need it. Because now the issue is that you could put anything in here, but it would not be validated by anything. OK in your case it does not matter, as only you work with this scheme, but it would be weird to me that you cannot just in a later time add a property to your schema, given that we use capnp for storage, It should be possible, no?

Reply:
I actually have no idea how the jsx schema handle these things.
Capnp required to follow a string field id instrumentation when you modify a schema. Which I'm quite sure the schema code doesn't handle properly in the capnp code generation.